tools/clang/unittests/HLSL/ValidationTest.cpp

```diff
@@ -4207,6 +4212,98 @@ TEST_F(ValidationTest, ValidateWithHash) {
   VERIFY_ARE_EQUAL(memcmp(Result, pHeader->Hash.Digest, sizeof(Result)), 0);
 }
 
+std::wstring GetEnvVarW(const std::wstring &VarName) {
+#ifdef _WIN32
+  if (const wchar_t *Result = _wgetenv(VarName.c_str()))
+    return std::wstring(Result);
+#endif
+  return std::wstring();
+}
```

Comment:
This pattern of quietly not doing anything on non-Win32 platforms doesn't seem right to me.  Couldn't we remove this function (and the tests that call it) from platforms that it doesn't apply to?